provider/src/test/kotlin/org/gradle/kotlin/dsl/integration/GradleKotlinDslIntegrationTest.kt

```diff
@@ -752,7 +750,7 @@ class GradleKotlinDslIntegrationTest : AbstractIntegrationTest() {
         """)
 
         val main = withBuildScript("""
-            apply { from("other.gradle.kts") }
```

Comment:
:o: So this will be a breaking change to remove this API?

Reply:
This PR doesn't remove any api, it adds only. This particular change is favoring the new api in a test, not working around any breaking change. I'm not sure what is your question, could you elaborate?

Reply:
I guess the ask is to make sure you leave tests in place that exercise the old API as well.